### DISCUSSION TITLE
Bug Report: can't build, does nothing.

### DISCUSSION THREAD

**Discussion by ROMZERD** [KadeArchive/Kade-Engine#608] [Q&A] [answered]
Windows Version: Windows 10 64 forgot what version it is

Kade Engine: Lastest as of last I checked (1.4.2)

I have done every step, I've installed all the dependencies and windows-only dependencies. Then clone Kade Engine, But when I open up cmd and run the command `lime build windows` it doesn't do anything. It just says nothing, I don't know what's happening, Can anyone help? I've linked an image below, Thanks!
![image](https://user-images.githubusercontent.com/84981768/120008213-595ef500-bfa0-11eb-9188-361afb225cd5.png)

**Comment by ROMZERD** [comment]
And yes, I have looked in the folder it's supposed to build in. I saw nothing ):

**Comment by ROMZERD** [comment]
I see that kade saw this and moved it to discussions, does anyone here have a solution to this?

**Reply by ghost** [reply]
Did you dowload a release?
Also, do not press any Ctrl key combinations with window selected
Remember: git clone

**Reply by ROMZERD** [reply]
no i didnt download the release, i downloaded the source
???
i cloned it using git, didnt i say that in my post?

**Reply by ghost** [reply]
Try PowerShell

**Reply by ROMZERD** [reply]
i did, same thing

**Reply by ghost** [reply]
Have you installed everything? Is the `lime` command set up?

**Comment by puyoxyz** [comment, marked as answer]
does `haxelib run lime build windows` work? if yes you set up lime wrong so do `haxelib run lime setup` again

this is a problem with your lime and not kade engine

**Reply by ROMZERD** [reply]
thanks so much it is working now!
